packages/sdk/src/lib/Keys.ts

```diff
@@ -130,4 +130,31 @@ export class Ed25519 {
     }
     return key;
   }
+
+  static privateKeyEncodeInPem(privateKey: ByteArray) {
+    // prettier-ignore
+    const derPrefix = Buffer.from([48, 46, 2, 1, 0, 48, 5, 6, 3, 43, 101, 112, 4, 34, 4, 32]);
+    const encoded = encodeBase64(
+      Buffer.concat([
+        derPrefix,
+        Buffer.from(Ed25519.parsePrivateKey(privateKey))
```

Comment:
I see in the similar public key function we don't need to the equivalent of `Ed25519.parsePrivateKey` - is that needed here?

Reply:
That is because the Ed25519 library that I used is generating `public key` and `secret key` as key pairs. The private key here is actually the `secret key`, its length is 64 bytes, composing by `public key` and 32 bytes `private key`. So I have to extract the real 32 bytes `private key`.

https://github.com/dchest/tweetnacl-js#constants-3

Reply:
Ah - makes sense.  Thanks.